discoverd/client/watch_test.go

```diff
@@ -19,16 +19,21 @@ type ClientSuite struct{}
 var _ = Suite(&ClientSuite{})
 
 func (s *ClientSuite) TestWatchReconnect(c *C) {
-	port, err := etcdrunner.RandomPort()
+	return // FIXME(benbjohnson): Fix discoverd watch reconnect.
```

Comment:
Does this FIXME need fixing before merging? If not, we should mark the test as skipped.